### ISSUE TITLE
Add a reset button

### ISSUE THREAD

**Issue by NathanBnm** [naaando/rush#14]
It can help if you don't want to restart the app. What do you think?

**Comment by naaando** [comment]
Yeah, I'm considering some UI changes for the next major release that I'll fix this.

**Comment by naaando** [comment]
I'm planning to break these sessions into tasks and possibly integrating it from a task app like Timetable or/and Planner.

**Comment by NathanBnm** [comment]
Maybe that could be a good idea
